tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs

```diff
@@ -1762,6 +1799,11 @@ private void SetupArchiveTournament(int tournamentId)
             _tournamentServiceMock.Setup(tr => tr.Archive(tournamentId));
         }
 
+        private void SetupUnArchiveTournament(int tournamentId)
```

Comment:
what is the difference from 1928?

@ViznyakS already asked that, btw

Reply:
Deprecated now.